internal/driver/interactive.go

```diff
@@ -218,7 +218,20 @@ func parseCommandLine(input []string) ([]string, variables, error) {
 	cmd, args := input[:1], input[1:]
 	name := cmd[0]
 
-	c := pprofCommands[cmd[0]]
+	c := pprofCommands[name]
+	if c == nil {
+		// Attempt splitting digits on abbreviated commands (eg top10)
+		for i := len(name); i > 0; i-- {
```

Comment:
would not a regexp like
re := regexp.MustCompile("[0-9]+$")
match := re.FindStringSubmatch(name)
(and then test match == nil for fail or match[0] for found number) be simpler in this case?